prac_06/guitars.py

```diff
@@ -0,0 +1,23 @@
+"""
+Write a program that uses the Guitar class
+and uses a list to store all the user's guitars (keep inputting until they enter a blank name),
+then print their details.
+"""
+from prac_06.guitar import Guitar
+
+my_guitars = []
+
+print("My guitars!")
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    my_guitars.append(Guitar(name, year, cost))
+    print(f"{name} ({year}) : ${cost:,.2f} added")
+    name = input("Name: ")
+print("These are my guitars:")
+for i, guitar in enumerate(my_guitars, 1):
+    vintage_status = ""
```

Comment:
can make the vintage_status into a single line of code
e.g. vintage_status = "(Vintage)" if guitar.is_vintage() else ""

Reply:
Was lines 20-23, not what its showing above

Reply:
Thank you for the feedback :)